tree.go

```diff
@@ -0,0 +1,130 @@
+package main
+
+import (
+	"fmt"
+	"path/filepath"
+
+	metadata "github.com/checkpoint-restore/checkpointctl/lib"
+	"github.com/checkpoint-restore/go-criu/v6/crit"
+	"github.com/checkpoint-restore/go-criu/v6/crit/images"
+	spec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/xlab/treeprint"
+)
+
+func renderTreeView(tasks []task) error {
+	for _, task := range tasks {
+		containerConfig, _, err := metadata.ReadContainerCheckpointConfigDump(task.outputDir)
+		if err != nil {
+			return err
+		}
+
+		specDump, _, err := metadata.ReadContainerCheckpointSpecDump(task.outputDir)
+		if err != nil {
+			return err
+		}
+
+		ci, err := getContainerInfo(task.outputDir, specDump, containerConfig)
+		if err != nil {
+			return err
+		}
+
+		archiveSizes, err := getArchiveSizes(task.checkpointFilePath)
+		if err != nil {
+			return err
+		}
+
+		tree := buildTree(ci, containerConfig, archiveSizes)
+
+		if mounts {
+			addMountsToTree(tree, specDump)
+		}
+
+		if stats {
+			dumpStats, err := crit.GetDumpStats(task.outputDir)
+			if err != nil {
+				return fmt.Errorf("failed to get dump statistics: %w", err)
+			}
+
+			addDumpStatsToTree(tree, dumpStats)
+		}
+
+		if psTree {
+			c := crit.New("", "", filepath.Join(task.outputDir, "checkpoint"), false, false)
+			psTree, err := c.ExplorePs()
+			if err != nil {
+				return fmt.Errorf("failed to get process tree: %w", err)
+			}
+
+			addPsTreeToTree(tree, psTree)
+		}
+
+		fmt.Printf("\nDisplaying container checkpoint tree view from %s\n\n", task.checkpointFilePath)
+		fmt.Println(tree.String())
+	}
+
+	return nil
+}
+
+func buildTree(ci *containerInfo, containerConfig *metadata.ContainerConfig, archiveSizes *archiveSizes) treeprint.Tree {
+	if ci.Name == "" {
+		ci.Name = "Container"
+	}
+	tree := treeprint.NewWithRoot(ci.Name)
+
+	tree.AddBranch(fmt.Sprintf("Image: %s", containerConfig.RootfsImageName))
+	tree.AddBranch(fmt.Sprintf("ID: %s", containerConfig.ID))
+	tree.AddBranch(fmt.Sprintf("Runtime: %s", containerConfig.OCIRuntime))
+	tree.AddBranch(fmt.Sprintf("Created: %s", ci.Created))
+	tree.AddBranch(fmt.Sprintf("Engine: %s", ci.Engine))
+
+	if ci.IP != "" {
+		tree.AddBranch(fmt.Sprintf("IP: %s", ci.IP))
+	}
+	if ci.MAC != "" {
+		tree.AddBranch(fmt.Sprintf("MAC: %s", ci.MAC))
+	}
+
+	tree.AddBranch(fmt.Sprintf("CHKPT Size: %s", metadata.ByteToString(archiveSizes.checkpointSize)))
```

Comment:
I believe the abbreviation "CHKPT Size" was initially used to fit within the table view. However, with tree / json view perhaps with can use "Checkpoint Size" instead?